extensions/wikia/DesignSystem/ds.built.js

```diff
@@ -0,0 +1,768 @@
+					dropdown.classList.remove(activeClass);
+				} else {
+					openDropdown(dropdown);
+				}
+			});
+		}
+
+		/* harmony default export */ __webpack_exports__["default"] = ({
+			hookDropdown: hookDropdown
+		});
+
+		/***/ }),
+	/* 2 */
+	/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+		"use strict";
+		Object.defineProperty(__webpack_exports__, "__esModule", { value: true });
+		/* harmony export (immutable) */ __webpack_exports__["init"] = init;
+		function init() {
+			console.log('hi from test-component');
+		}
+
+		/* harmony default export */ __webpack_exports__["default"] = ({
+			init: init
+		});
+
+		/***/ }),
+	/* 3 */
+	/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+		"use strict";
+		/* harmony import */ var __WEBPACK_IMPORTED_MODULE_0__element_matches__ = __webpack_require__(4);
+		/* harmony export (immutable) */ __webpack_exports__["a"] = closestElement;
+
+		/**
+		 * Get the closest ancestor element that matches a give selector
+		 * Similar to jQuery's $.fn.closest
+		 * @param {Element} elem
+		 * @param {string} selector
+		 * @returns {Element|null}
+		 */
+		function closestElement(elem, selector) {
+			if (__webpack_require__.i(__WEBPACK_IMPORTED_MODULE_0__element_matches__["a" /* default */])(elem, selector)) {
+				return elem;
+			}
+
+			if (!elem.parentElement) {
+				return null;
+			}
+
+			return closestElement(elem.parentElement, selector);
+		}
+
+		/***/ }),
+	/* 4 */
+	/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+		"use strict";
+		/* harmony export (immutable) */ __webpack_exports__["a"] = elementMatches;
+		/**
+		 * Is this element queryable by the given selector?
+		 * Similar to jQuery's $.fn.is
+		 *
+		 * @param {Element} element
+		 * @param {string} selector
+		 * @returns {boolean}
+		 */
+		function elementMatches(element, selector) {
+			var selectorElements = (element.document || element.ownerDocument).querySelectorAll(selector);
+			var matches = false;
+
+			[].forEach.call(selectorElements, function (selectorElement) {
+				if (selectorElement === element) {
+					matches = true;
+				}
+			});
+
+			return matches;
+		}
+
+		/***/ }),
+	/* 5 */
+	/***/ (function(module, exports, __webpack_require__) {
+
+		exports = module.exports = __webpack_require__(6)(undefined);
+		// imports
+
+
+		// module
+		exports.push([module.i, ".wds-dropdown {\n\t--content-z-index: 1;\n\n\tdisplay: inline-block;\n\tposition: relative;\n}\n\n.wds-dropdown__toggle {\n\tcursor: pointer;\n\tposition: relative;\n\t/* dropdown content has shadow that should be hidden below the toggle*/\n\tz-index: calc(var(--content-z-index) + 1);\n}\n\n.wds-dropdown__content {\n\tdisplay: none;\n\tleft: 0;\n\tposition: absolute;\n\ttop: 100%;\n\tz-index: var(--content-z-index);\n}\n\n.wds-dropdown.wds-is-active .wds-dropdown__content {\n\t display: inline-block;\n }\n", ""]);
+
+		// exports
+
+
+		/***/ }),
+	/* 6 */
+	/***/ (function(module, exports) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "exports". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)